docs/fargate.md

```diff
@@ -115,6 +120,135 @@ fi
+        region region-code
+        log_group_name my-logs
+        log_stream_prefix from-fluent-bit-
+        log_retention_days 60
+        auto_create_group true
+```
+
+For more information on this refer to [fargate-logging](https://docs.aws.amazon.com/eks/latest/userguide/fargate-logging.html)
+
+### Cloudwatch logs collection
+
+After setting up fluent-bit to forward logs to cloudwatch, the next step is to setup and enable cloudwatch collection
+
+This involves the following:
+
+- [Setup Cloudwatch authentication](#authenticate-with-cloudwatch)
+- [Enable Clouwatch collection](#enable-cloudwatch-collection-eks-fargate-only)
+
+#### Authenticate with Cloudwatch
+
+To configure the service account to use an IAM role (for authentication), follow the steps below
+
+- Set your AWS account ID to an environment variable
+- Set your cluster's OIDC identity provider to an environment variable. Replace `my-cluster` with the name of your cluster
+- Set the namespace and the service account name
+
+```bash
+account_id=$(aws sts get-caller-identity --query "Account" --output text)
+oidc_provider=$(aws eks describe-cluster --name my-cluster --region $AWS_REGION --query "cluster.identity.oidc.issuer" --output text | sed -e "s/^https:\/\///")
+export namespace=default
+export service_account=my-service-account
+```
+
+Set the following trust relationship
+
+```bash
+cat >trust-relationship.json <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Federated": "arn:aws:iam::$account_id:oidc-provider/$oidc_provider"
+      },
+      "Action": "sts:AssumeRoleWithWebIdentity",
+      "Condition": {
+        "StringEquals": {
+          "$oidc_provider:aud": "sts.amazonaws.com",
+          "$oidc_provider:sub": "system:serviceaccount:$namespace:$service_account"
+        }
+      }
+    }
+  ]
+}
+EOF
+```
+
+After these variables have been set and the above trush relationship is defined, the next step is to create an IAM role and attach it to a
+policy.
+
+```bash
+aws iam create-role --role-name my-role --assume-role-policy-document file://trust-relationship.json --description "my-role-description"
+aws iam attach-role-policy --role-name my-role --policy-arn=arn:aws:iam::$account_id:policy/my-policy
+```
+
+#### Enable cloudwatch collection (EKS fargate only)
+
+Update the `user-values.yaml` to add the following annotation to the service account. Replace `account-id` and `my-role` with the
+appropriate values
+
+```yaml
+sumologic:
+  logs:
+    collector:
+      otelcloudwatch:
+        enabled: true
+        roleArn: arn:aws:iam::account-id:role/my-role
+        # cloudwatch receiver configuration: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/awscloudwatchreceiver
+        awscloudwatch:
+          region: us-east-2
+          logs:
+            poll_interval: 1m
+            groups:
+              autodiscover:
+                limit: 10
```

Comment:
We should hide the configuration if possible. I don't feel this one is user-friendly and our customers usually are not interested in implementation details (which I consider exposing direct configuration)

Reply:
We could point the customer to the cloudwatch receiver configuration. That way we could just tell the customer that we support cloudwatch receiver configuration as is. Allowing this flexibility might work to our benefit as different customers may have different configuration requirements for cloudwatch receivers.